include/zephyr/bluetooth/hci_types.h

```diff
@@ -37,6 +37,14 @@ extern "C" {
 #define BT_HCI_SYNC_HANDLE_INVALID 0xffff
 #define BT_HCI_PAWR_SUBEVENT_MAX 128
 
+/* Bluetooth spec 5.2 Vol 4, Part E - 5.4.3 HCI Synchronous Data Packets */
+struct bt_hci_sco_hdr {
+	uint16_t  conn_handle;
+	uint8_t  len;
+} __packed;
```

Comment:
Please follow the same struct alignment style as elsewhere in the headers, so that the struct member names are aligned. That means two spaces after `uint8_t` but only one after `uint16_t`.

Reply:
changed